Source/WebCore/inspector/agents/InspectorCSSAgent.cpp

```diff
@@ -491,6 +491,9 @@ Protocol::ErrorStringOr<std::tuple<RefPtr<JSON::ArrayOf<Protocol::CSS::RuleMatch
     if (!element)
         return makeUnexpected(errorString);
 
+    if (!element->isConnected())
```

Comment:
NIT: I wish we had something like `Element::canGetStyle` or `Element::hasComputedStyle` so that we can unify all these `isConnected` that are actually related.  I suppose that's what `isConnected` does, but I'd hate for it to change in the future and this (or something else) to get mistakenly left behind.